Comparison Engine/Comparison Engine/Comparison Engine.csproj

```diff
@@ -7,4 +7,9 @@
     <UseWindowsForms>true</UseWindowsForms>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="3.1.8" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
```

Comment:
Are we using both of these packages? Nes lygtais minėjai, kad nesigavo su Entities, tai if there's no features using them, they shouldn't be committed just yet. In that case would need to uninstall the package at the time of commit (maybe there's some less ape method of doing this, but I don't know of it yet)